### PR TITLE
[PLAT-545] Move Nitro related tools in a separate directory

### PR DESCRIPTION
**Pull request by raoulstrackx**
The rust-sgx repo contains a lot of vme/nitro related crates. These are all in the same directory. It is cleaner to place them in separate crates.
This PR does not perform any functional changes.

### CODE REVIEW

**Review by Pagten — APPROVED** [review]

**Comment by Pagten** [comment]
bors d+

File: Cargo.toml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 [workspace]
 members = [
+    "fortanix-vme/aws-nitro/eif-tools",
```
Comment (jethrogb) [inline Cargo.toml:3]: It's nitro enclaves. Nitro is something else.

Comment (raoulstrackx) [reply Cargo.toml]: Yes you have a point that `aws-nitro-enclaves` is a more suitable name.

**Pull request merged**